### PR TITLE
[#43184] Change wording: use add when adding storages to a project

### PR DESCRIPTION
**Pull request by fraber**
https://community.openproject.org/work_packages/43184

### CODE REVIEW

File: modules/storages/config/locales/en.yml [hunk 1]
```diff
@@ -46,7 +46,7 @@ en:
     page_titles:
       project_settings:
         index: "File storages available in this project"
-        new: "Enable a file storage for this project"
+        new: "Add a file storage for this project"
```
Comment (wielinde) [inline modules/storages/config/locales/en.yml]:
```suggestion
        new: "Add a file storage to this project"
```

**Comment by wielinde** [comment]
@fraber could you please fix the base branch for this PR?

**Comment by ali-alhaidary** [comment]
@fraber  shouldn't it be in main.json?

**Review by apfohl — APPROVED** [review]

**Comment by apfohl** [comment]
@fraber You still have the failing tests. When they a green, it's good to go IMO.

**Comment by fraber** [comment]
> @fraber You still have the failing tests. When they a green, it's good to go IMO.

@apfohl, please have a look at the failed feature spec:

- spec/features/work_packages/work_package_workflow_form_spec.rb:93

This is far outside the scope of this PR, seems to be a flickering spec...

**Comment by apfohl** [comment]
> > @fraber You still have the failing tests. When they a green, it's good to go IMO.
> 
> @apfohl, please have a look at the failed feature spec:
> 
>     * spec/features/work_packages/work_package_workflow_form_spec.rb:93
> 
> 
> This is far outside the scope of this PR, seems to be a flickering spec...

Fine for me.

**Pull request merged**